ecs-agent/logger/log.go

```diff
@@ -44,18 +44,34 @@ const (
 	DEFAULT_TIMESTAMP_FORMAT                 = time.RFC3339
 	DEFAULT_MAX_FILE_SIZE            float64 = 10
 	DEFAULT_MAX_ROLL_COUNT           int     = 24
+	DEFAULT_LOGTO_STDOUT                     = true
 )
 
+// Because timestamp format will be called in the custom formatter
+// for each log message processed, it should not be handled
+// with an explicitly write protected configuration.
+var timestampFormat = DEFAULT_TIMESTAMP_FORMAT
+
+// logLevels is the mapping from ECS_LOGLEVEL to Seelog provided levels.
+var logLevels = map[string]string{
+	"debug": "debug",
+	"info":  "info",
+	"warn":  "warn",
+	"error": "error",
+	"crit":  "critical",
+	"none":  "off",
+}
+
 type logConfig struct {
-	RolloverType    string
-	MaxRollCount    int
-	MaxFileSizeMB   float64
-	logfile         string
-	driverLevel     string
-	instanceLevel   string
-	outputFormat    string
-	timestampFormat string
-	lock            sync.Mutex
+	RolloverType  string
+	MaxRollCount  int
+	MaxFileSizeMB float64
+	logfile       string
+	driverLevel   string
+	instanceLevel string
+	outputFormat  string
+	logToStdout   bool
```

Comment:
Is this a new feature?

Reply:
Could you clarify? logToStdout is not necessarily new. It just makes the console receiver optional that is on by default.

Reference https://github.com/cihub/seelog/wiki/Receiver-reference

Reply:
This is a new setting on the logger. 

> Adds a new variable to the Config, logToStdout that will decide whether we should duplicate output to stdout or not. Currently, we are outputting to a rolling log file as well as stdout. The latter should be optional.

That sounds like a new feature to me. Why do we need this in this PR?

Reply:
When consuming this change, I noticed that all logs were being logged to my desired config file as well as journal logs. This is not ideal and introduces clutter to our log streams. So users should be able to configure where their logs land, if anywhere.

Reply:
Discussed offline that this is a new feature that we want the logger to have.